### PR TITLE
Cluster session updates db bootstrap

### PR DESCRIPTION
**Pull request by tock-ibm**

### CODE REVIEW

File: internal/replica_set.go [hunk 1]
```diff
@@ -53,6 +54,38 @@ func (r ReplicaSet) SortByRole() {
...
+		configReplicaSet = append(configReplicaSet, &config.Replica{
+			ID:       v.Id,
+			Endpoint: v.URL.String(),
+		})
+	}
+
+	return configReplicaSet
+}
+
+// ToReplicaMap returns map of ID[URL] that corresponds the ReplicaSet.
```
Comment (gennadylaventman) [inline internal/replica_set.go:74]: `map if [id]url` or `id->url`

File: pkg/bcdb/db_test.go [hunk 2]
```diff
@@ -0,0 +1,80 @@
+// Copyright IBM Corp. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package bcdb
+
+import (
+	"fmt"
```
Comment (gennadylaventman) [inline pkg/bcdb/db_test.go:7]: goimport

File: pkg/bcdb/session.go [hunk 3]
```diff
@@ -11,6 +11,7 @@ import (
 	"encoding/base64"
 	"encoding/json"
 	"fmt"
+	"github.com/hyperledger-labs/orion-sdk-go/pkg/config"
```
Comment (gennadylaventman) [inline pkg/bcdb/session.go:14]: goimport

File: internal/replica_set.go (hunk 1 above)
Comment (tock-ibm) [reply internal/replica_set.go:74]: done

File: pkg/bcdb/db_test.go (hunk 2 above)
Comment (tock-ibm) [reply pkg/bcdb/db_test.go:7]: done

File: pkg/bcdb/session.go (hunk 3 above)
Comment (tock-ibm) [reply pkg/bcdb/session.go]: done

File: pkg/bcdb/db.go [hunk 4]
```diff
@@ -314,9 +321,31 @@ func (b *bDB) Session(cfg *config.SessionConfig) (DBSession, error) {
 		return nil, errors.Wrap(err, "cannot update the replica set and signature verifier")
 	}
 
+	b.updateReplicaMap(session.replicaSet.ToReplicaMap())
+
 	return session, nil
 }
 
+func (b *bDB) getReplicaMap() map[string]*url.URL {
```
Comment (gennadylaventman) [inline pkg/bcdb/db.go:329]: I didn't find where this function used...

Comment (tock-ibm) [reply pkg/bcdb/db.go:329]: done

**Pull request merged**